package-lock.json

```diff
@@ -974,6 +974,33 @@
           "requires": {
             "@types/estree": "*"
           }
+        },
+        "periscopic": {
+          "version": "2.0.3",
+          "resolved": "https://registry.npmjs.org/periscopic/-/periscopic-2.0.3.tgz",
```

Comment:
update `code-red` as well? 
with https://github.com/Rich-Harris/code-red/pull/63, `periscopic`, `estree` and `is-reference` version should be the same then?